scraper/src/config/version.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
+
+__version__ = "0.12.9"
+
+
+def qualified_version() -> str:
+    """Get the qualified version of this module."""
+
+    return f"Meilisearch Docs-scraper (v{__version__})"
```

Comment:
```suggestion
    return f"Meilisearch DocsScraper (v{__version__})"
```

Reply:
Why is this better than `docs-scraper` ? The other repo's seem to follow the `docs-scraper` ways no?  https://github.com/meilisearch/docs-searchbar.js/blob/c53e287bf9709b87661d7e4a2327fc830e16a78b/src/lib/agents.js#L3-L7

Maybe my example one is wrong though

Reply:
It's a matter of taste actually! 

I just don't like to mix kebab case with pascal case at the same time! It should be one or another:

`DocsScraper` or `docs-scraper`